config/initializers/node-orm2.js

```diff
@@ -19,9 +19,6 @@ export default {
         adapter.db = db;
         let models = container.lookupAll('model');
         let adapterModels = mapValues(models, (Model) => {
-          if (Model.hasOwnProperty('abstract') && Model.abstract) {
-            return;
-          }
```

Comment:
@acorncom made convincing arguments in Gitter for `abstract` being part of the Denali model API. 👍  to include it. Of course, it will be up to each ORM adapter to figure out how to implement abstract base classes sanely against their ORM.